packages/editor/src/components/visual-editor/index.js

```diff
@@ -358,8 +358,6 @@ function VisualEditor( {
 		! isMobileViewport &&
 		// Dsiable resizing in zoomed-out mode.
 		! isZoomedOut;
-	const shouldIframe =
-		! disableIframe || [ 'Tablet', 'Mobile' ].includes( deviceType );
```

Comment:
When I first commented, I wasn’t understanding you meant to remove the conditions here but it does make sense to me now. Still, it looks like the tablet and mobile conditions got left out of the post editor’s `useShouldIframe` hook. I tested and the device previews weren’t working from a non-iframe starting point.

Reply:
Sorry, in my repeated commits I seem to have forgotten to update the `useShouldIframe` hook. It should work as expected now.